tmt/checks/journal.py

```diff
@@ -388,10 +323,18 @@ def after_test(
         logger: tmt.log.Logger,
     ) -> list[CheckResult]:
         if not invocation.guest.facts.has_systemd:
-            return [CheckResult(name='journal', result=ResultOutcome.SKIP)]
+            return [
+                CheckResult(
+                    name='journal', result=ResultOutcome.SKIP, note=['systemd not available']
+                )
+            ]
 
         if not invocation.is_guest_healthy:
-            return [CheckResult(name='journal', result=ResultOutcome.SKIP)]
+            return [
+                CheckResult(
+                    name='journal', result=ResultOutcome.SKIP, note=['guest is not healthy']
```

Comment:
Same as above, we have nicer notes, `note=hints_as_notes('guest-not-healthy'),`

Reply:
resolved